tools/donate-cpu-server.py

```diff
@@ -1,5 +1,6 @@
 
 # Server for 'donate-cpu.py'
```

Comment:
Perhaps we should add `#!/usr/bin/env python3` (and make the file executable if it already isn't)?

Reply:
Then the test script calling it can be changed to use `../donate-cpu-server.py ...` and we wont have to update it when it is time for python 4. 😀

Reply:
Yes, I thought about adding the shebang also. I think it is a good thing to add it. And I will change the test script. Thanks for reminding me :)

Reply:
Done now, thanks!